src/SprykerSdk/Zed/SprykGui/Communication/Form/BaseSprykForm.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Zed\SprykGui\Communication\Form;
+
+use Spryker\Zed\Kernel\Communication\Form\AbstractType;
+use SprykerSdk\Zed\SprykGui\Communication\Form\Type\ArgumentCollectionType;
+use SprykerSdk\Zed\SprykGui\Communication\Form\Type\OutputChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * @method \SprykerSdk\Zed\SprykGui\SprykGuiConfig getConfig()
+ * @method \SprykerSdk\Zed\SprykGui\Business\SprykGuiFacadeInterface getFacade()
+ * @method \SprykerSdk\Zed\SprykGui\Communication\SprykGuiCommunicationFactory getFactory()
+ */
+class BaseSprykForm extends AbstractType
+{
+    /**
+     * @param mixed[] $arguments
+     *
+     * @return mixed[]
+     */
+    protected function getRelevantArguments(array $arguments): array
+    {
+        $filteredArguments = [];
+
+        foreach ($arguments as $argumentName => $argumentDefinition) {
+            if (
+                in_array($argumentName, ['module', 'dependentModule', 'organization'])
```

Comment:
Can we please use true for 3rd arg for stringish comparisons?

Reply:
Sure, fixed.